wallet/src/Pos/Wallet/Web/State/Storage.hs

```diff
@@ -359,59 +466,76 @@ addRemovedAccount addrMeta@CWAddressMeta{..} = do
         accInfo . aiAddresses        . at cwamId .= Nothing
         accInfo . aiRemovedAddresses . at cwamId ?= AddressInfo addrMeta key
 
+-- | Update account metadata.
 setAccountMeta :: AccountId -> CAccountMeta -> Update ()
 setAccountMeta accId cAccMeta = wsAccountInfos . ix accId . aiMeta .= cAccMeta
 
+-- | Update wallet metadata.
 setWalletMeta :: CId Wal -> CWalletMeta -> Update ()
 setWalletMeta cWalId cWalMeta = wsWalletInfos . ix cWalId . wiMeta .= cWalMeta
 
+-- | Change wallet ready status.
 setWalletReady :: CId Wal -> Bool -> Update ()
 setWalletReady cWalId isReady = wsWalletInfos . ix cWalId . wiIsReady .= isReady
 
+-- | Update wallet's last password change time.
 setWalletPassLU :: CId Wal -> PassPhraseLU -> Update ()
 setWalletPassLU cWalId passLU = wsWalletInfos . ix cWalId . wiPassphraseLU .= passLU
 
+-- | Declare that given wallet has been synced with blockchain up to block with given
+-- header hash.
 setWalletSyncTip :: CId Wal -> HeaderHash -> Update ()
 setWalletSyncTip cWalId hh = wsWalletInfos . ix cWalId . wiSyncTip .= SyncedWith hh
 
+-- | Update transaction meta data for given wallet id and transaction id.
 addWalletTxHistory :: CId Wal -> CTxId -> CTxMeta -> Update ()
 addWalletTxHistory cWalId cTxId cTxMeta =
     wsTxHistory . at cWalId . non' _Empty . at cTxId ?= cTxMeta
 
+-- | Update meta data for multiple wallet transactions at once.
 setWalletTxHistory :: CId Wal -> [(CTxId, CTxMeta)] -> Update ()
 setWalletTxHistory cWalId cTxs = mapM_ (uncurry $ addWalletTxHistory cWalId) cTxs
 
--- FIXME: this will be removed later (temporary solution)
+-- | Set meta data for transaction only if it hasn't been set already.
+-- FIXME: this will be removed later (temporary solution) (not really =\)
```

Comment:
:/

Reply:
_I'm not going to see anime with drama anymore_
_I have enough of it in code_ :droplet: